server/admin/admin_server_test.go

```diff
@@ -561,8 +562,17 @@ func TestServer_UpdateTree(t *testing.T) {
 		Description:     "Brand New Tree Desc",
 		StorageSettings: settings,
 		MaxRootDuration: ptypes.DurationProto(2 * time.Nanosecond),
+		PrivateKey: func() *any.Any {
```

Comment:
Suggestion: `PrivateKey: testonly.MustMarshalAny(t, &empty.Empty{}),`

(where `testonly` is [github.com/google/trillian/testonly](https://github.com/google/trillian/blob/master/testonly/marshalling.go#L41))

Reply:
Done.